purchase_report_date_format/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Report Date Format",
+    "category": "Reporting",
+    "version": "14.0.1.0.1",
```

Comment:
New modules should start with version `*.*.1.0.0`
```suggestion
    "version": "14.0.1.0.0",
```

see https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers:
> For example: 12.0.1.0.0 is expected for the first release of an 12.0 module.

Reply:
:+1: